docs/antrea-network-policy.md

```diff
@@ -243,7 +244,8 @@ spec:
         - protocol: TCP
           port: 6379
       name: AllowFromFrontend
-      enableLogging: false
+      logSettings:
```

Comment:
I think we can remove all "logSettings" in other examples, as we have a separate one for it, and the filed is not set by default

Reply:
Done.